Admin/AdminInterface.php

```diff
@@ -1093,10 +1093,13 @@ public function getDashboardActions();
      /**
       * Configure buttons for an action.
       *
+      * @param array  $actions
       * @param string $action
       * @param object $object
+      *
+      * @return array
       */
-     public function configureActionButtons($action, $object = null);
+     public function configureActionButtons($actions, $action, $object = null);
```

Comment:
Oh didn't see it was a parameter addition. Got `$actions` and `$action`? Looks weird.